service/resource/endpoint/resource.go

```diff
@@ -53,27 +53,49 @@ func (r *Resource) Underlying() framework.Resource {
 	return r
 }
 
-func toPod(v interface{}) (*apiv1.Pod, error) {
+func getAnnotations(pod apiv1.Pod, ipAnnotationName string, serviceAnnotationName string) (ipAnnotationValue string, serviceAnnotationValue string, err error) {
```

Comment:
I reordered functions here to be in alphabetical order, so the diff looks weird.